extensions/danbooru/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "danbooru",
+  "title": "Danbooru",
+  "description": "Search the anime image board Danbooru!",
+  "icon": "extension-icon.png",
+  "author": "abel_roinsard",
+  "categories": [
+    "Media",
+    "Fun",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-danbooru",
+      "title": "Search Danbooru",
+      "subtitle": "Get in on the anime fun!",
+      "description": "Fetch images from the Danbooru website according to your preferences.",
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "username",
+          "description": "The name of your Danbooru account. Found above 'statistics' on your account page. Not your user ID!",
+          "type": "textfield",
+          "required": true,
+          "title": "Danbooru Username",
+          "placeholder": "Username"
+        }, {
+          "name": "apiKey",
+          "description": "An API key is needed in order to perform requests. It is found in your Danbooru profile page. This extensions needs at least the following permissions: posts:index, posts:show and posts:show_seq",
+          "type": "textfield",
```

Comment:
```suggestion
          "type": "password",
```

Reply:
added in 58d8a571c3f8a59ef93e56544237d6c4d1484cfe. Is the difference with `textfield` related to encryption or something ? I did not really notice anything different in my testing.

Reply:
Yes it is 🙂